src/main/kotlin/com/team4099/robot2022/config/Constants.kt

```diff
@@ -114,4 +114,30 @@ object Constants {
     const val THROTTLE_DEADBAND = 0.05
     const val TURN_DEADBAND = 0.05
   }
+
+  object Feeder {
+
+    //feeder_power temp
+    const val FEEDER_POWER = 1.0
+    const val FLOOR_MOTOR_ID = 61
+    const val VERTICAL_MOTOR_ID = 62
+    const val TAB = "Feeder"
+
+    //temp values
+    const val TOP_DIO_PIN = 1
+    const val BOTTOM_DIO_PIN = 1
```

Comment:
we can set one to 0 and the other to 1 as we won't be using any other DIOs

Reply:
once done with this, the branch looks fine to me so in terminal execute the following command:
./gradlew :spotlessApply 
this allows it to build properly